runtime/s390x.S

```diff
@@ -891,6 +891,8 @@ CFI_STARTPROC
         CFI_OFFSET(14, -168)
         ENTER_FUNCTION
         LEA_VAR(caml_array_bound_error_asm, ADDITIONAL_ARG)
+    /* Note the following jumps in the middle of caml_c_call, since stack
+       has already been adjusted. */
         brcl    15, LBL(caml_c_call)
```

Comment:
The comment is right, but the original code is strange. I believe it could just `LEA_VAR` then jump to `GCALL(caml_c_call)`, which would then do the stack adjustment.  (This is what the arm64 port does, and it's a good reference.)  But "strange code" doesn't mean "wrong code", so no change is warranted here.

Reply:
For GDB backtraces, perhaps.  For OCaml's own backtraces, it should not make a difference.